src/java/org/jivesoftware/openfire/labelling/Clearance.java

```diff
@@ -0,0 +1,14 @@
+package org.jivesoftware.openfire.labelling;
+
+import org.dom4j.Element;
+
+/**
+ * Created by dwd on 16/03/17.
+ */
+public class Clearance {
+    Element element;
+
+    Clearance(Element element) {
+        this.element = element;
+    }
+}
```

Comment:
This is an odd wrapper class. Shouldn't the class state be more standard Java types (String, int, boolean)? Perhaps also add `hashCode` and `equals` implementations?